config.json

```
@@ -892,6 +892,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 3
+      },
+      {
+        "slug": "ledger",
+        "name": "Ledger",
+        "uuid": "fb9758db-359a-4839-b37e-7a462906846a",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

```suggestion
        "difficulty": 4
```
This is not an easy exercise